content/ai/concepts/neural-networks/neural-networks.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'Neural Networks'
+Description: 'A neural network is a method in artificial intelligence that teaches computers to process data in a way that is inspired by the human brain.'
+Subjects:
+  - 'Machine Learning'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Machine Learning'
+  - 'Neural Networks'
+  - 'Weight & Bias'
+  - 'Deep Learning'
+  - 'Perceptrons'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/data-science'
+---
+
+**Artificial neural networks (ANN)** are a method in artificial intelligence which teaches computers to process data in a similar way such as the human brain. 
+
+## What Is An Artificial Neural Network (ANN)
+
+- An ANN a type of machine learning process, called deep learning, that uses interconnected nodes or neurons in a layered structure that resembles the human brain.
+- An ANN is one of the main tools used for machine learning, it is a brain-inspired system used to replicate the way that humans learn.
+
+### The Process
+
+- Neural networks are trained on specific data which then get broken down into the input layer (where information is fed into) then to the hidden layer (where weight and bias are calculated from the input layer) and lastly the output layer (where information is output after algorithms are processed from the hidden and input layers) via perceptrons (similar to human neurons but for machines to process, store, calculate and move data forward).
+
+## Relation To AI
+
+### AI
+
+- AI works on a collection of technologies which allow the computer to sense, learn, reason & act, it powers algorithms which are instructions encoded in computer programs.
+- AI works because of machine learning which is equipped with algorithms and vast amounts of datasets trained to recognize patterns and solve complex problems.
+
+### ANN
+
+- Neural networks are the core of machine learning algorithms. A neural network is trained on specific datasets from which it extracts the information and passes it through different layers searching for patterns.
+- Once a neural network is trained on a dataset it can start to encounter new scenarios and start making predictions based off that previous dataset. 
```

Comment:
```suggestion
Once a neural network is trained on a dataset, it can start to encounter new scenarios and start making predictions based on that previous dataset. 
```

Reply:
done